packages/circuit-ui/components/Toggle/Toggle.module.css

```diff
@@ -95,7 +95,7 @@
 
 .label {
   display: block;
-  margin-left: var(--cui-spacings-kilo);
+  padding-left: var(--cui-spacings-kilo);
```

Comment:
Using `padding` instead of `margin` removes a gap between the toggle and the label where users weren't able to interact with the toggle.